spring-boot-jib/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+server.port= 8086
+server.servlet.context-path=/spring-boot-jib
```

Comment:
Is this custom context path necessary? e.g. I think without it, we'd just do localhost:8086/greeting and we'd be fine right?

Reply:
Removed the file itself, since both properties are not needed. Actually, I added it so that the port and context are very independent of all other applications in source code. But, I guess nobody is going to deploy source code on a single machine, so we can delete it.